examples/tools/autotools/create_your_first_package_windows.rst

```diff
@@ -57,8 +57,11 @@ execute the build in it.
 
   It is not necessary, and in fact it is not recommended for most cases to be already running inside an ``msys2``
   terminal. Conan will automatically run the build subprocess for autotools in the defined bash shell.
-
-  If already running in a bash shell, it is necessary to activate the ``tools.microsoft.bash:active=True`` conf.
+  
+  If already running in a bash shell, it is necessary to activate the
```

Comment:
Not sure this is the best place for this clarification, this example is not very visible.
Maybe add it to the ``global.conf`` description itself?

Then, it might not be fully clear, maybe turn it around: "if running in a msys2 subsystem that uses a /bin/sh or bash shell as default subprocess shell, it is necessary to activate ``active=True`` ..."

Reply:
Yes, I agree, the description in the source code would be a better place, I'll do a PR in the Conan client.